tasks/monitoring/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - import_tasks: all_in_one.yml
-  when: bbb_monitoring_all_in_one_enable | bool
+  when: bbb_monitoring_all_in_one_enable | bool or bbb_monitoring_external | bool
```

Comment:
Maybe it's a good idea to splitout the external tasks further and include them separately when bbb_monitoring_external | bool or bbb_monitoring_all_in_one_enable | bool